### ISSUE TITLE
IF you mention THIS issue, your DAD is GAY!!!

### ISSUE THREAD

**Issue by markass-brownie-not-real-i-am-fake** [WinampDesktop/winamp#2697]
But not as gay as my app, which you can check out at [panels.art](https://panels.art)! - Markass Brownie

**Comment by 000yesnt** [comment]
wallapr

**Comment by markass-brownie-not-real-i-am-fake** [comment]
> wallapr

I agree, but not as much as i agree with my app, which you can check out at [panels.art](https://panels.art)! - Markass Brownie

**Comment by hboloid** [comment]
im gay

**Comment by garfielfandjohn** [comment]
> > wallapr
> 
> I agree, but not as much as i agree with my app, which you can check out at [panels.art](https://panels.art)! - Markass Brownie

MARKASS!! HOW DID YOU RISE FROM THE DEAD!!!

**Comment by markass-brownie-not-real-i-am-fake** [comment]
> #2697

Your dad is gay, and he discovered his love for homosexuality because of me. Also make sure to check out my app, which you can check out at [panels.art](https://panels.art)! - Markass Brownie

**Comment by markass-brownie-not-real-i-am-fake** [comment]
> > > wallapr
> > 
> > 
> > I agree, but not as much as i agree with my app, which you can check out at [panels.art](https://panels.art)! - Markass Brownie
> 
> MARKASS!! HOW DID YOU RISE FROM THE DEAD!!!
![popsicle](https://github.com/user-attachments/assets/d19f461f-50ce-47dc-a3c0-2586df6f4cee)
